packages/state-transition/src/block/index.ts

```diff
@@ -69,8 +69,8 @@ export function processBlock(
     processBlobKzgCommitments(externalData);
     // Only throw preDeneb so beacon can also sync/process blocks optimistically
```

Comment:
Maybe update comment here?

```suggestion
    // Only throw preData so beacon can also sync/process blocks optimistically
```